ui/StatusQ/src/StatusQ/Core/Utils/RecursiveStackView.qml

```diff
@@ -0,0 +1,49 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+/*!
+   \qmltype RecursiveStackView
+   \inherits StackView
+   \inqmlmodule StatusQ.Core.Utils
+
+   \brief A utility wrapper over StackView allowing to compose StackViews in
+    a convenient way.
+*/
+StackView {
+    // this stack of nested one if currentItem is RecursiveStackView, recusively
+    readonly property RecursiveStackView topLevelStack: {
+        if (currentItem instanceof RecursiveStackView)
+            return currentItem.topLevelStack
+
+        return this
+    }
+
+    // busy flag of this stack of nested one if currentItem is
+    // RecursiveStackView, recusively
```

Comment:
I'm not getting the meaning of `... of this stack of nested one if currentItem is RecursiveStackView, recusively`, which you repeat multiple times in this file 😵‍💫 
Is there an `or` missing somewhere? 🤔

Reply:
yeah, there is a typo, `of` -> `or`, fixed